contributing.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to Shadcn Organization
+
+Thank you for considering contributing to the Shadcn organization! Before you start contributing, please take a moment to read through this guide to understand how you can contribute effectively.
+
+## Introduction
+
+Welcome to Shadcn! We are a community focused on creating re-usable components using Radix UI and Tailwind CSS. However, it's important to clarify that Shadcn is not a traditional component library that you install as a dependency via npm. Instead, it's a collection of re-usable components that you can freely copy and paste into your projects.
+
+### What do we mean by "not a component library"?
+
+When we say Shadcn is not a component library, we mean that you won't find it published on npm or distributed as a standalone package. Instead, Shadcn provides you with a set of code snippets and components that you can cherry-pick for your projects. You have the freedom to copy and paste these components directly into your application's codebase.
+
+### How to use Shadcn components?
+
+1. **Pick the Components You Need:** Explore the Shadcn repository to find the components that best suit your project's requirements.
+
+2. **Copy and Paste:** Simply copy the code of the selected component(s) from the Shadcn repository.
+
+3. **Customize as You Like:** The copied code is now yours to modify and customize to fit your specific needs. You have full control over how these components look and behave in your application.
+
+4. **Reference and Learn:** Use Shadcn as a reference and learning resource to build your own component libraries. Feel free to contribute back to Shadcn if you create improvements or new components that might benefit others in the community.
+
+## Getting Started
+
+To start contributing to Shadcn, follow these steps:
+
+1. **Fork the Repository:** Click the "Fork" button on the Shadcn repository's GitHub page to create your own copy of the repository.
+
+2. **Clone Your Fork:** Clone your forked repository to your local machine using Git:
+
+    ```bash
+    git clone https://github.com/your-username/shadcn.git
+    ```
+
+3. **Create a New Branch:** Create a new Git branch for your contribution:
+
+    ```bash
+    git checkout -b feature/my-new-component
+    ```
+
+4. **Make Changes:** Make the necessary changes or additions to the components you want to contribute. Ensure your code follows the project's coding standards and practices.
+
+5. **Commit Changes:** Commit your changes with a clear and descriptive commit message:
+
+    ```bash
+    git commit -m "Add: My awesome new component"
+    ```
+
+6. **Push Changes:** Push your branch to your forked repository on GitHub:
+
+    ```bash
+    git push origin feature/my-new-component
```

Comment:
The correct command to push from a branch would be: "git push --set-upstream origin feature/my-new-component"